packages/expo-dev-menu/expo-dev-menu.podspec

```diff
@@ -1,5 +1,31 @@
 require 'json'
 
+# reanimated 2
+
+reactVersion = '0.0.0'
+
+begin
+  reactVersion = JSON.parse(File.read(File.join(__dir__, "..", "react-native", "package.json")))["version"]
```

Comment:
would be great to have a more robust method of finding the package.json, like using require.resolve